format/testfiles/test.rego.formatted

```diff
@@ -19,10 +19,7 @@ foo[x] {
 	g(x, "foo") = z
 }
 
-globals = {
-	"foo": "bar",
-	"fizz": "buzz",
-}
+globals = {"fizz": "buzz", "foo": "bar"}
```

Comment:
Hmm not sure what to make of this one. I hadn't expected changes to how source files are formatted coming from this 🤔